src/Drivers/BcryptLegacy.ts

```diff
@@ -0,0 +1,83 @@
+/*
+ * @adonisjs/hash
+ *
+ * (c) Harminder Virk <virk@adonisjs.com>
+ * and
+ * (c) Evgeniy Ryumin <cmp08@ya.ru>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+/// <reference path="../../adonis-typings/hash.ts" />
+import { BcryptLegacyContract, BcryptLegacyConfig } from '@ioc:Adonis/Core/Hash'
+import bcrypt from 'bcryptjs'
```

Comment:
Why not use the `bcrypt` library instead `bcryptjs`?

Reply:
I saw that Adonis V4 use `bcrypt` package, do the two hashes have the same format?